pkg/services/publicdashboards/api/api_test.go

```diff
@@ -424,6 +428,25 @@ func TestApiCreatePublicDashboard(t *testing.T) {
 			SaveDashboardErr:     nil,
 			User:                 userNoRBACPerms,
 			ShouldCallService:    false,
+			JsonBody:             `{ "isPublic": true }`,
+		},
+		{
+			Name:                 "returns 400 when uid is invalid",
+			ExpectedHttpResponse: http.StatusBadRequest,
+			publicDashboard:      &PublicDashboard{IsEnabled: true},
```

Comment:
this could be nil, since it's the return of the service create method that will never be reachable

Reply:
Sorry, what could be nil? publicDashboard?

Reply:
yes!